plugins/inventory/terraform_state.py

```diff
@@ -26,11 +26,23 @@
     required: true
     type: str
     choices: [ cloud.terraform.terraform_state ]
-  backend_config:
+  backend_type:
     description:
-      - A Terraform backend configuration to an existing state file.
+      - The Terraform backend type from which the state file will be retrieved.
+      - Possible values include C(s3), C(remote), C(azurerm), C(local), C(consul), C(cos), C(gcs), C(http)
```

Comment:
The way this is written it suggests that these are the only possible values. I think I would just remove this line, especially since we aren't checking the provided type against any list.

Reply:
updated